manage.py

```diff
@@ -28,15 +28,15 @@ def run(self, remaining):
         if len(remaining) == 0:
             print('path to ccextractor is missing')
             return 1
+
+        path_to_ccex = remaining[0]
+        print('path to ccextractor: ' + str(path_to_ccex))
+
+        if update_expected_results(path_to_ccex):
```

Comment:
It's a bit of nitpicking, but this if-else can (just as the previous one) be simplified further :)